core/src/main/java/com/rim/IceField/Game.java

```diff
@@ -117,4 +118,184 @@ public static void Turn(PlayerBase player) throws Exception {
+                player.useItem("Shovel");
+                System.out.println("Action accepted!");
+            } else if (text.get(1).equals("diving suit")) {
+                player.useItem("Diving Suit");
+                System.out.println("Action accepted!");
+            }
+        }
+        else if(text.get(0).equals("pick"))
+        {
+            System.out.println("Would you like to pick " + player.currentIceberg.getItem() + "press 1 for yes, 2 for no");
+            Scanner input = new Scanner(System.in);
+            int y = input.nextInt();
+            if(y == 1) {
+                try {
+                    player.pickItem();
+                    System.out.println("Action accepted!");
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
+            }
+        }
+
+        else if(text.get(0).equals("apply skill"))
+        {
+            if(text.get(1).equals("NORTH"))
+            {
+                try {
+                    player.useSkill(this.getMap(), "north");
+                    System.out.println("Action accepted!");
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
+            }
+            else if(text.get(1).equals("SOUTH"))
+            {
+                try {
+                    player.useSkill(this.getMap(), "south");
+                    System.out.println("Action accepted!");
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
+            }
+            else if(text.get(1).equals("WEST"))
+            {
+                try {
+                    player.useSkill(this.getMap(), "west");
+                    System.out.println("Action accepted!");
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
+            }
+            else if(text.get(1).equals("EAST"))
+            {
+                try {
+                    player.useSkill(this.getMap(), "east");
+                    System.out.println("Action accepted!");
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
+            }
+
+        }
+        //Will distinguish between players based on their unique ID
+        else if(text.get(0).equals("save"))
+        {
+                player.SavePlayer(text.get(1), players);
+            System.out.println("Action accepted!");
+        }
+
+        else if(text.get(0).equals("remove snow"))
+        {
+            player.removeSnow();
+            System.out.println("Action accepted!");
+        }
+
+        else if(text.get(0).equals("fire"))
+        {
+            GameOver();
+            System.out.println("Action accepted!");
+        }
+
+        else if(text.get(0).equals("inv"))
+        {
+            System.out.println("Action accepted!");
+            for(int i = 0 ; i < player.getInventory().getItems().size(); i++)
+            {
```

Comment:
you can print a message like:
You have:
-food x4
-rope x1
-diving suit x1

Reply:
ok